let a = 12 ;
let b = 1 ; 

let sum = a+b ;

try {
    console.log(sum*x);
} catch (error) {
    console.log(error.name);
    console.log(error.message);
    console.log(error.stack);
}